### DISCUSSION TITLE
How do we use the included 404 error page?

### DISCUSSION THREAD

**Discussion by ghost** [adityatelange/hugo-PaperMod#956] [Q&A]
Hey, how do we use the included 404 page? When I go to a non-existent page, I get the default Nginx 404 error. Thanks.

**Comment by adityatelange** [comment]
https://gohugo.io/templates/404/#automatic-loading

**Comment by jasonculligan** [comment]
The link doesn't actually explain how to leverage the PaperMod's 404.html page.  I can't point nginx to that page as it's not true html.

```html
{{- define "main" }}
<div class="not-found">404</div>
{{- end }}{{/* end main */ -}}
```

**Reply by adityatelange** [reply]
Run 'hugo' in your site site directory. Go to public and you'll find 404.html
